components/admin_console/jobs/table.jsx

```diff
@@ -286,7 +286,7 @@ class JobTable extends React.PureComponent {
             );
         }
 
-        const date = new Date(millis);
+        const date = new Date(new Date(millis).toUTCString().substr(0, 25));
```

Comment:
I think we should use a localized version of this date display using moment.js, which is already a JS package we ship with.